input/system/tembo/system.go

```diff
@@ -0,0 +1,165 @@
+	Metric Metric        `json:"metric"`
+	Value  []interface{} `json:"value"`
+}
+
+type Metric struct {
+	Name                  string `json:"__name__"`
+	Endpoint              string `json:"endpoint"`
+	Instance              string `json:"instance"`
+	Job                   string `json:"job"`
+	MetricsPath           string `json:"metrics_path"`
+	Namespace             string `json:"namespace"`
+	Node                  string `json:"node"`
+	PersistentVolumeClaim string `json:"persistentvolumeclaim"`
+	Service               string `json:"service"`
+}
+
+// GetSystemState - Gets system information for a Tembo Cloud instance
+func GetSystemState(config config.ServerConfig, logger *util.Logger) (system state.SystemState) {
+	headers := map[string]string{
+		"Authorization": "Bearer " + config.TemboAPIToken,
+		"accept":        "application/json",
+	}
+
+	client := http.Client{}
+
+	// Get CPU usage percentage
+	query := "sum(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{ namespace=\"" + config.TemboNamespace + "\"}) / sum(kube_pod_container_resource_requests{job=\"kube-state-metrics\",  namespace=\"" + config.TemboNamespace + "\", resource=\"cpu\"})"
+	cpuUsage, err := getFloat64(query, "https://api.data-1.use1.tembo.io/"+config.TemboNamespace+"/metrics/query?query=", client, headers)
+	if err != nil {
+		logger.PrintError("Tembo/System: Encountered error when getting CPU info %v\n", err)
+		return
+	}
+
+	system.CPUStats = make(state.CPUStatisticMap)
+	system.CPUStats["all"] = state.CPUStatistic{
+		DiffedOnInput: true,
+		DiffedValues: &state.DiffedSystemCPUStats{
+			UserPercent: cpuUsage,
+		},
+	}
+
+	// Get total memory
+	query = "sum(max by(pod) (kube_pod_container_resource_requests{job=\"kube-state-metrics\", namespace=\"" + config.TemboNamespace + "\", resource=\"memory\"}))"
+	memoryTotalBytes, err := getUint64(query, "https://api.data-1.use1.tembo.io/"+config.TemboNamespace+"/metrics/query?query=", client, headers)
+	if err != nil {
+		logger.PrintError("Tembo/System: Encountered error when getting memory info %v\n", err)
+		return
+	}
+
+	// Get available memory
+	query = "sum(max by(pod) (kube_pod_container_resource_requests{job=\"kube-state-metrics\", namespace=\"" + config.TemboNamespace + "\", resource=\"memory\"})) - sum(container_memory_working_set_bytes{job=\"kubelet\", metrics_path=\"/metrics/cadvisor\", namespace=\"" + config.TemboNamespace + "\",container!=\"\", image!=\"\"})"
+	memoryAvailableBytes, err := getUint64(query, "https://api.data-1.use1.tembo.io/"+config.TemboNamespace+"/metrics/query?query=", client, headers)
+	if err != nil {
+		logger.PrintError("Tembo/System: Encountered error when getting memory info %v\n", err)
+		return
+	}
+
+	system.Memory.TotalBytes = memoryTotalBytes
+	system.Memory.AvailableBytes = memoryAvailableBytes
+	system.Memory.FreeBytes = memoryAvailableBytes
+
+	// Get disk capacity
+	//TODO(ianstanton) Check if volume claim names differ in cases like HA
```

Comment:
As discussed, would be good to double check before merging (and if not an issue to remove the TODO)

Reply:
We do have separate PVCs when HA is enabled. I'd like to handle this as a follow-up if it's alright with you. Example:
```
⨠ kubectl get po -n org-demo-inst-pganalyze-test 
NAME                                                      READY   STATUS      RESTARTS   AGE
org-demo-inst-pganalyze-test-1                            1/1     Running     0          11h
org-demo-inst-pganalyze-test-2                            0/1     Running     0          102s
org-demo-inst-pganalyze-test-2-snapshot-recovery-s6dfb    0/1     Completed   0          119s
org-demo-inst-pganalyze-test-pganalyze-86f568fb59-k66lm   1/1     Running     0          3d9h


 ⨠ kubectl get pvc -n org-demo-inst-pganalyze-test 
NAME                             STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
org-demo-inst-pganalyze-test-1   Bound    pvc-2793b508-f3f5-44f2-8b50-dd3b3d387745   10Gi       RWO            gp3-enc        27d
org-demo-inst-pganalyze-test-2   Bound    pvc-3428da10-62a4-4670-b4ba-5b9e9e963ba5   10Gi       RWO            gp3-enc        3m34s

```